test/fixtures/sample-openapiv3.0.0-schema.json

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
Maybe having the entire openapi v3.0.0 spec is a bit too much. Let me know if you want me to just use a simple one that uses draft-04.
I used this because it's the use case I wanted to cover (and I think supporting openapi v3.0.0 is very important as there are tools that are unable to upgrade to 3.1).